modules/seqexec/server/src/main/scala/seqexec/server/ghost/GHOSTController.scala

```diff
@@ -39,37 +39,61 @@ final case class GHOSTController[F[_]: Sync](ghostClient: GHOSTClient[F],
     def cfg[P: Show](paramName: String, paramVal: P) =
       Configuration.single(s"${ifuNum.ifuStr}.$paramName", paramVal)
```

Comment:
this is unrelated to the code but I'd expect the config names to be prefixed like:
`s"ghost:${ifName...`

Reply:
It's in the `ifuStr:`
```
 sealed abstract class IFUNum(val ifuNum: Int) {
    val ifuStr: String = s"ghost:cc:cu:ifu$ifuNum"
  }
```

Reply:
Everything only has to do with IFUs right now, so I threw it in there. I could pull it out, though, if that seems better.

Reply:
ah ok, thanks for the clarification